### PR TITLE
Error instead of commit on bad format

### PR DESCRIPTION
**Pull request by fredrik-bakke**
Turns out it's not as easy as expected to set up invisible autoformatting. Thus, instead, this PR makes it so the workflow just blocks ill-formatted PRs. Contributors will then have to install an autoformatter (e.g. Prettier) locally to fix the issue.

### CODE REVIEW

**Comment by emilyriehl** [comment]
Hi @fizruk @fredrik-bakke I've lost track of what's up with autoformating. Now whenever I merge something I'm getting an error message about autoformating failure. Can someone explain the current status?

**Comment by fredrik-bakke** [comment]
This pr makes it so other prs are blocked until they are properly formatted. The recommended procedure should then be for contributors to set up autoformatting in their ide. Given that you agree we should have this feature/rule of course.

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -1,30 +1,19 @@
-name: Code autoformatting
+name: File formatting
 
 on:
-  push:
+  pull_request:
```
Comment (fizruk) [inline .github/workflows/format.yml:4]: Perhaps, we still want to check on pushes to `main`?

**Review by fizruk — APPROVED** [review]

**Comment by fizruk** [comment]
@fredrik-bakke can you please update/fix the formatting (I have updated the branch, and it needs some formatting now)?

@fredrik-bakke @emilyriehl I suggest that we merge this before #103 (which supersedes #100) and help @cesarbm03 with his PR in case of conflicts.

**Comment by fredrik-bakke** [comment]
I'll be at my computer in a couple of hours. I can fix it then

**Comment by fredrik-bakke** [comment]
Okay, this one should be ready for merging again now. Apologies if this has caused problems for anyone. In #94, I'm adding a remark to the documentation that we enforce autoformatting rules and that contributors should install an autoformatter.

**Review by fizruk — APPROVED** [review]

**Pull request merged**